modules/javafx.controls/src/main/java/javafx/scene/control/skin/SpinnerSkin.java

```diff
@@ -261,13 +262,25 @@ private boolean isIncDecKeyEvent(KeyEvent ke) {
      *                                                                         *
      **************************************************************************/
 
+    @Override
+    public void install() {
+        getChildren().add(textField);
```

Comment:
`dispose()` method removes 3 children, whereas `install()` method adds only 1 child. Is this intentional?

Reply:
good question!  yes, it is intentional - the textField, unlike the buttons, comes from the control, so when we change skins the old skin removes it in its dispose(), followed by the new skin adding it back in install(), as it cannot do that in the constructor.

added a comment in the code to clarify